derive/src/filter.rs

```diff
@@ -199,6 +199,32 @@ pub fn recursive_filter_fn(fields: &[IdentTypeTuple]) -> Result<TokenStream, cra
                             condition = condition.add(Column::#column_enum_name.is_null())
                         }
                     }
+
+                    if let Some(is_not_null_value) = seaography::FilterTrait::is_not_null(#column_name) {
+                        if is_not_null_value {
+                            condition = condition.add(Column::#column_enum_name.is_not_null())
+                        }
+                    }
```

Comment:
If one can supply a `false` value to `is_null` and `is_not_null`. Then, we better handle it properly.

Reply:
Fixed.